getting_started/macos_apple_silicon.md

```diff
@@ -16,7 +16,15 @@ which includes the Roc compiler and various helpful utilities.
 1. Untar the archive:
 
     ```sh
-    roc_nightly-darwin_apple_silicon-<VERSION>.tar.gz
+    mkdir roc_nightly-darwin_apple_silicon-<VERSION>
+    tar xf roc_nightly-darwin_apple_silicon-<VERSION>.tar.gz --directory roc_nightly-darwin_apple_silicon-<VERSION>
+    ```
+
+1. Install llvm 13 and manually symlink the package (note that you'll want to remove this symlink before adding the llvm package, if you need that for another project):
+
+    ```sh
+    brew install llvm@13
+    ln -s /opt/homebrew/Cellar/llvm@13/13.0.1_1/ /opt/homebrew/opt/llvm
```

Comment:
Hi. Thank you for updating the instructions!

Instead of creating the symlink, I would suggest to run the following command:

```console
export DYLD_LIBRARY_PATH=$(brew --prefix llvm@13)/lib:$DYLD_LIBRARY_PATH
```

This will set the library path only for the current terminal session.

Reply:
oops. I overlooked this comment.

https://github.com/roc-lang/roc/pull/4032#issuecomment-1251055064
> I've changed the nighlty build server so that produced nightlies now look for llvm in `/opt/homebrew/opt/llvm@13`.

Please ignore my suggestion above.